### ISSUE TITLE
Doesn't pull any images

### ISSUE THREAD

**Issue by Lumineir** [sal0max/muzei-nationalgeographic#17]
Expected behavior: pulls the daily image from national geographic

Actual behavior: does not download any image

**Comment by echeoquehaii** [comment]
Also experiencing this, the list stays black and no image is pulled

**Comment by sal0max** [comment]
Yep, the API seems to not be available any more: https://www.nationalgeographic.co.uk/page-data/photo-of-day/page-data.json

TBH, I haven't used my plugin for quite some time any more, myself. I'm currently lacking the time to investigate a fix. So, if anyone will provide a Pull Request, I'd appreciate.

Edit: Seems, they completely killed the photo of the day on the NatGeo UK. Can't find it at all on their website. In that case, I can't do much.

**Comment by echeoquehaii** [comment]
Thanks for looking into this @sal0max. Too bad they discontinued it!

Do you have any idea if they have an open Git somewhere or where we could ask to bring it back?

Thanks for your work until now though, it was a pleasure to use this addon!
